lib/cloud/aws/policy_statements.go

```diff
@@ -406,6 +406,9 @@ func StatementAccessGraphAWSSync() *Statement {
 			"s3:ListBucket",
 			"s3:GetBucketLocation",
 			"s3:GetBucketTagging",
+			"s3:GetBucketPolicyStatus",
+			"s3:GetBucketAcl",
```

Comment:
Do we need to update this doc file as well?
https://github.com/gravitational/teleport/blob/b6ffcee62146e16741458970bad27d7e0de69935/docs/pages/admin-guides/teleport-policy/integrations/aws-sync.mdx?plain=1#L154-L158